src/oemof/solph/_plumbing.py

```diff
@@ -43,7 +54,7 @@ def sequence(iterable_or_scalar):
     if isinstance(iterable_or_scalar, abc.Iterable) and not isinstance(
         iterable_or_scalar, str
     ):
-        return iterable_or_scalar
+        return np.array(iterable_or_scalar)
```

Comment:
There is a problem with this, as `sequence` is used to create sequences of practically infinite length from scalar values.  If you return a numpy.array instead, the following will happen:

```python
# current implementation
s = sequence(sequence([5]))
s[0] = 5
s[5] = 5

# your suggestion
s = sequence(sequence([5]))
s[0] = 5
s[5] -> out of range
```